apps/extension/src/App/LockWrapper/LockWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+import { Icon, IconName } from "@anoma/components";
+import { ButtonContainer, LockExtensionButton } from "./LockWrapper.components";
+import { LockKeyRingMsg } from "background/keyring";
+import { Ports } from "router";
+import { Status } from "App/App";
+import { TopLevelRoute } from "App/types";
+import { ExtensionRequester } from "extension";
+
+type Props = {
+  requester: ExtensionRequester;
+  setStatus: (status?: Status) => void;
```

Comment:
nit: left a comment about the ambiguous naming of this 
>nit: This went in already earlier but now that it is being used outside of this file, I feel now that it is being used outside of this file, this name is even more ambiguous as one cannot easily read the code to get the whole context of "what status this is".

https://github.com/anoma/namada-interface/pull/84/files#r992316411